webdriver/client.py

```diff
@@ -385,12 +377,13 @@ def send_command(self, method, uri, body=None):
 
         url = urlparse.urljoin("session/%s/" % self.session_id, uri)
         response = self.send_raw_command(method, url, body)
+        value = response.body["value"]
 
-        rv = response.body["value"]
-        if not rv:
-            rv = None
+        if response.status != 200:
```

Comment:
If we want to keep `send_command` agnostic of the actual command being sent, then this approach would prevent invocation of `/status` during active sessions. That seems like a surprising restriction, but I'm also reluctant to extend `send_command` with branches for different commands. What do you think?